includes/acl/compression/impl/clip_context.h

```diff
@@ -137,6 +137,17 @@ namespace acl
 			transform_metadata* metadata				= nullptr;
 			uint32_t* leaf_transform_chains				= nullptr;
 
+#ifdef ACL_COMPRESSION_OPTIMIZED
+
+			struct transform_link
+			{
+				uint32_t parent_transform_index;
+				uint32_t child_transform_index;
+			};
+			transform_link* transform_links				= nullptr;	// 1 per transform, sorted in reverse order, for compact_constant_streams and quantization_context.adjusted_shell_distances.
```

Comment:
Reserve order = children first?

Reply:
Fair point, I can be clearer about what "reverse order" means.